.tool-versions

```diff
@@ -0,0 +1,2 @@
+elixir 1.12.0-otp-24
+erlang 24.3.4
```

Comment:
In `.travis.yml`:
- `elixir` is `1.13.4`
- `otp_release` is `25.0`

But also, is it worth committing these files?
And can their values just be obtained from files in the project itself, like `.travis.yml`?

Reply:
We've been storing the tool-versions on redwood to make local setup a bit easier, but that's a good point, I can remove that.

We're using gh actions where the tool-versions can be re-used which can help with the amount of places.